domain/build.gradle.kts

```
@@ -34,10 +36,19 @@ dependencies {
     implementation(libs.androidx.core.ktx)
     implementation(libs.androidx.appcompat)
     implementation(libs.material)
+    implementation(project(":mediaservice"))
```

다른 모듈과 통일되도록 가장 위에 projects로 추가부탁드립니다.

```suggestion
    implementation(projects.mediaservice)
```